nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -142,6 +142,11 @@ pt-services.clipcat.enable).
 
 - the `mingw-64` package has been upgraded from 6.0.0 to 9.0.0
 
+- The following Visual Studio Code extensions were renamed to keep the naming convention uniform.
+  - `bbenoist.Nix` -> `bbenoist.nix`
+  - `CoenraadS.bracket-pair-colorizer` -> `coenraads.bracket-pair-colorizer`
+  - `golang.Go` -> `golang.go`
```

Comment:
This is a useless release note. Just add aliases for them.

Reply:
Should I open another PR to fix this as this has been merged already?

Reply:
You can leave the release notes like they are. I think adding aliases is worth it anyway. Best would it be if they are gated behind allowAliases.